@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -183,6 +184,166 @@ export interface BaseStyle {
+ *             "attr": {
+ *                 "color": "#$2fff",
+ *                 "lineWidth": "^${3:1}"
+ *             }
+ *         }
+ *     },
+ *     {
+ *         "label": "New dashed-line",
+ *         "description": "Add a New 'dashed-line' Styling Rule",
+ *         "body": {
+ *             "technique": "dashed-line",
+ *             "when": "$1",
+ *             "attr": {
+ *                 "color": "#$2fff",
+ *                 "lineWidth": "^${3:1}"
+ *             }
+ *         }
+ *     },
+ *     {
+ *         "label": "New fill",
+ *         "description": "Add a New 'fill' Styling Rule",
+ *         "body": {
+ *             "technique": "fill",
+ *             "when": "$1",
+ *             "attr": {
+ *                 "color": "#$2fff"
+ *             }
+ *         }
+ *     },
+ *     {
+ *         "label": "New text",
+ *         "description": "Add a New 'text' Styling Rule",
+ *         "body": {
+ *             "technique": "text",
+ *             "when": "$1",
+ *             "attr": {
+ *                 "scale": "^$2"
+ *             }
+ *         }
+ *     },
+ *     {
+ *         "label": "New labeled-icon",
+ *         "description": "Add a New 'labeled-icon' Styling Rule",
+ *         "body": {
+ *             "technique": "labeled-icon",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New line-marker",
+ *         "description": "Add a New 'line-marker' Styling Rule",
+ *         "body": {
+ *             "technique": "line-marker",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New line",
+ *         "description": "Add a New 'line' Styling Rule",
+ *         "body": {
+ *             "technique": "line",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New segments",
+ *         "description": "Add a New 'segments' Styling Rule",
+ *         "body": {
+ *             "technique": "segments",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New standard",
+ *         "description": "Add a New 'standard' Styling Rule",
+ *         "body": {
+ *             "technique": "standard",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New extruded-line",
+ *         "description": "Add a New 'extruded-line' Styling Rule",
+ *         "body": {
+ *             "technique": "extruded-line",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New extruded-polygon",
+ *         "description": "Add a New 'extruded-polygon' Styling Rule",
+ *         "body": {
+ *             "technique": "extruded-polygon",
+ *             "when": "$1",
+ *             "attr": {}
+ *         }
+ *     },
+ *     {
+ *         "label": "New none",
```

Comment:
Do we really need snippets for everything, including unnecessary things like none?

Reply:
In fact, there are two cases where `"none"` technique can be useful: 
 * to add when rule or some attrs that will be inherited by nested styles;
 * used together with `"when": "..."` and `"final": true`, to completely disable some features

Reply:
I'd say that the snippets are nice, but it would be nice to take a bit more care with them and put some meaningful initial values. Some like the ones for ``standard`` are a bit too empty...